Changes

```diff
@@ -1,6 +1,22 @@
 Revision history for Zonemaster Backend.
```

Comment:
Should we be consistent, or it does not matter?

Reply:
Updated the heading to be consistent with your CLI PR.